src/api/ApiCall.cpp

```diff
@@ -0,0 +1,26 @@
+#include "ApiCall.hh"
+
+#include "api\IServer.hh"
+
+#include "services\ServiceLocator.hh"
+
+namespace ra {
+namespace api {
+
+static ra::api::IServer& Server() noexcept
+{
+    return ra::services::ServiceLocator::GetMutable<ra::api::IServer>();
+}
+
+Login::Response Login::Request::Call() const noexcept
+{
+    return Server().Login(*this);
```

Comment:
The `Call` method implementation redirects the request parameters to the current `IServer` implementation. By putting the implementation in this file, the individual API header files only have to know about their singular API. `IServer` knows about all APIs.